src/Clue.js

```diff
@@ -1,6 +1,6 @@
 class Clue {
   constructor(data) {
-    this.data = data;
+    this.data = data.data;
```

Comment:
`data.data` is a little weird…can you rename your parameter to something more specific/semantic? Would `clues.data` make sense?